<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>health_home</class>
 <widget class="QWidget" name="health_home">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>940</width>
    <height>599</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QFrame" name="frame_body">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>391</width>
     <height>241</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#frame_body {
    background-color: rgb(255, 255, 255); /* 设置背景颜色为白色 */
    border: 1px solid rgb(200, 200, 200); /* 设置边框颜色 */
    border-radius: 10px;                  /* 设置圆角半径 */
}
</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="horizontalSpacing">
     <number>5</number>
    </property>
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="minimumSize">
         <size>
          <width>41</width>
          <height>31</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>41</width>
          <height>31</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="textFormat">
         <enum>Qt::AutoText</enum>
        </property>
        <property name="pixmap">
         <pixmap resource="image.qrc">:/icons/widgets/body.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_bodytitle">
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>-1</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">
QLabel {
	background-color: rgb(255, 255, 255);
    color: rgb(72,61,139);  /* 字体颜色 */
    font-size: 16px;  /* 字体大小为 16 像素 */
    
}</string>
        </property>
        <property name="text">
         <string>身体参数</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="minimumSize">
         <size>
          <width>61</width>
          <height>41</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>61</width>
          <height>41</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="textFormat">
         <enum>Qt::AutoText</enum>
        </property>
        <property name="pixmap">
         <pixmap resource="image.qrc">:/icons/widgets/heart_rate.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_heartRate">
        <property name="font">
         <font>
          <pointsize>22</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>显示心率：</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetFixedSize</enum>
      </property>
      <item>
       <widget class="QLabel" name="label">
        <property name="minimumSize">
         <size>
          <width>61</width>
          <height>41</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>61</width>
          <height>41</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="textFormat">
         <enum>Qt::AutoText</enum>
        </property>
        <property name="pixmap">
         <pixmap resource="image.qrc">:/icons/widgets/bllod.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_bloodOxygen">
        <property name="font">
         <font>
          <family>宋体</family>
          <pointsize>22</pointsize>
          <bold>false</bold>
          <kerning>true</kerning>
         </font>
        </property>
        <property name="mouseTracking">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>血氧浓度</string>
        </property>
        <property name="textFormat">
         <enum>Qt::AutoText</enum>
        </property>
        <property name="scaledContents">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="3" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="minimumSize">
         <size>
          <width>61</width>
          <height>41</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>61</width>
          <height>41</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="textFormat">
         <enum>Qt::AutoText</enum>
        </property>
        <property name="pixmap">
         <pixmap resource="image.qrc">:/icons/widgets/people_temp.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_peopleTemp">
        <property name="font">
         <font>
          <pointsize>22</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>显示体温</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>90</y>
     <width>511</width>
     <height>371</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#frame_2 {
    background-color: rgb(255, 255, 255); /* 设置背景颜色为白色 */
    border: 1px solid rgb(200, 200, 200); /* 设置边框颜色 */
    border-radius: 10px;                  /* 设置圆角半径 */
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_12">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label_12">
        <property name="minimumSize">
         <size>
          <width>41</width>
          <height>31</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>41</width>
          <height>31</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="textFormat">
         <enum>Qt::AutoText</enum>
        </property>
        <property name="pixmap">
         <pixmap resource="image.qrc">:/icons/widgets/env.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_bodytitle_2">
        <property name="font">
         <font>
          <family>华文行楷</family>
          <pointsize>-1</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">
QLabel {
	background-color: rgb(255, 255, 255);
    color: rgb(72,61,139);  /* 字体颜色 */
    font-size: 16px;  /* 字体大小为 16 像素 */
    
}</string>
        </property>
        <property name="text">
         <string>环境参数</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="4" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_9">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label_9">
        <property name="minimumSize">
         <size>
          <width>61</width>
          <height>41</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>61</width>
          <height>41</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="textFormat">
         <enum>Qt::AutoText</enum>
        </property>
        <property name="pixmap">
         <pixmap resource="image.qrc">:/icons/widgets/db.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_db">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>显示db</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_14">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="label_6">
          <property name="minimumSize">
           <size>
            <width>61</width>
            <height>41</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>61</width>
            <height>41</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 255);</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="textFormat">
           <enum>Qt::AutoText</enum>
          </property>
          <property name="pixmap">
           <pixmap resource="image.qrc">:/icons/widgets/CO2.png</pixmap>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_CO2">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 255);</string>
          </property>
          <property name="text">
           <string>显示CO2</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="label_8">
          <property name="minimumSize">
           <size>
            <width>61</width>
            <height>41</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>61</width>
            <height>41</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 255);</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="textFormat">
           <enum>Qt::AutoText</enum>
          </property>
          <property name="pixmap">
           <pixmap resource="image.qrc">:/icons/widgets/PM25.png</pixmap>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_PM25">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 255);</string>
          </property>
          <property name="text">
           <string>显示PM2.5</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_13">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="label_5">
          <property name="minimumSize">
           <size>
            <width>61</width>
            <height>41</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>61</width>
            <height>41</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 255);</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="textFormat">
           <enum>Qt::AutoText</enum>
          </property>
          <property name="pixmap">
           <pixmap resource="image.qrc">:/icons/widgets/TVOC.png</pixmap>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_TVOC">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 255);</string>
          </property>
          <property name="text">
           <string>显示TVOC</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="label_7">
          <property name="minimumSize">
           <size>
            <width>61</width>
            <height>41</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>61</width>
            <height>41</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 255);</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="textFormat">
           <enum>Qt::AutoText</enum>
          </property>
          <property name="pixmap">
           <pixmap resource="image.qrc">:/icons/widgets/HCHO.png</pixmap>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_HCHO">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 255);</string>
          </property>
          <property name="text">
           <string>显示甲醛</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="3" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_15">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_11">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="label_11">
          <property name="minimumSize">
           <size>
            <width>61</width>
            <height>41</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>61</width>
            <height>41</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 255);</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="textFormat">
           <enum>Qt::AutoText</enum>
          </property>
          <property name="pixmap">
           <pixmap resource="image.qrc">:/icons/widgets/temp.png</pixmap>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_temp">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 255);</string>
          </property>
          <property name="text">
           <string>显示环境湿度：</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_10">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="label_10">
          <property name="minimumSize">
           <size>
            <width>61</width>
            <height>41</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>61</width>
            <height>41</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 255);</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="textFormat">
           <enum>Qt::AutoText</enum>
          </property>
          <property name="pixmap">
           <pixmap resource="image.qrc">:/icons/widgets/humi.png</pixmap>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_humi">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 255);</string>
          </property>
          <property name="text">
           <string>显示环境湿度：</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QFrame" name="frame_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>911</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#frame_3 {
    background-color: rgb(255, 255, 255); /* 设置背景颜色为白色 */
    border: 1px solid rgb(200, 200, 200); /* 设置边框颜色 */
    border-radius: 10px;                  /* 设置圆角半径 */
}
</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QFrame" name="frame_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>340</y>
     <width>391</width>
     <height>251</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#frame_4 {
    background-color: rgb(255, 255, 255); /* 设置背景颜色为白色 */
    border: 1px solid rgb(200, 200, 200); /* 设置边框颜色 */
    border-radius: 10px;                  /* 设置圆角半径 */
}
</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QFrame" name="frame_5">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>470</y>
     <width>511</width>
     <height>121</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#frame_5 {
    background-color: rgb(255, 255, 255); /* 设置背景颜色为白色 */
    border: 1px solid rgb(200, 200, 200); /* 设置边框颜色 */
    border-radius: 10px;                  /* 设置圆角半径 */
}
</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="image.qrc"/>
 </resources>
 <connections/>
</ui>
